tests/nthash.cpp

```diff
@@ -315,5 +279,46 @@ main()
     TEST_ASSERT_EQ(i, 3);
   }
 
+  {
+    std::cerr << "Testing rolling spaced seeds vs masked hashing" << std::endl;
+
+    std::string seq = "AACGTGACTACTGACTAGCTAGCTAGCTGATCGT";
+    std::vector<std::string> seeds = { "111111111101111111111",
+                                       "110111010010010111011" };
+    unsigned k = seeds[0].length();
+    unsigned h = 2;
+
+    std::queue<uint64_t> masked_hashes;
+    for (unsigned i = 0; i <= seq.size() - k; i++) {
+      for (std::string seed : seeds) {
+        uint64_t fk = btllib::ntf64(seq.data() + i, k);
```

Comment:
Is it possible to test this functionality through the SeedNtHash class? I think the code would be easier to maintain that way as SeedNtHash is what the users will see and should use, whereas the low level nthash functions might change at any point.

Reply:
This test is to make sure that SeedNtHash generates the same output as old ntHash's approach to spaced seed hashing. I think we can keep it for now to find any compatibility issues, but I'll remove it when I'm refactoring the code.